server/utils/outOfServiceBedUtils.ts

```diff
@@ -111,7 +111,11 @@ export const outOfServiceBedActions = (
         {
           items: [
             {
-              text: 'Update record',
+              text: 'Cancel out of service bed',
```

Comment:
Should there be any check on permissions here? I guess to be done when a new, separate `cancel OOSB` permission is introduced?

Reply:
And it's there now!